src/vs/workbench/contrib/format/browser/formatActionsMultiple.ts

```diff
@@ -127,7 +127,8 @@ class DefaultFormatter extends Disposable implements IWorkbenchContribution {
 		const picks = formatter.map((formatter, index) => {
 			return <IIndexedPick>{
 				index,
-				label: formatter.displayName || formatter.extensionId || '?'
+				label: formatter.displayName || formatter.extensionId || '?',
+				description: formatter.extensionId!.value
```

Comment:
Maybe better `description: formatter.extensionId && formatter.extensionId.value` A formatter should always have an extension id but must not have one...

Reply:
I just kind of assumed the `!.` TypeScript operator worked like [`&.` in Ruby](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/), it seems like this is a better equivalent for `&.` anyway.